.github/workflows/aether_observatory.yml

```diff
@@ -0,0 +1,13 @@
+name: aether_observatory
+
+on:
+  push:
+
+jobs:
+  ruby:
+    uses: powerhome/github-actions-workflows/.github/workflows/ruby-gem.yml@main
+    with:
+      package: ${{ github.workflow }}
+      workdir: "packages/${{ github.workflow }}"
+      ruby: '["3.0","3.1","3.2","3.3"]'
```

Comment:
Most of our libraries are testing things in all these rubies, but in reality all we need is 3.0 and 3.3.